documents/FAQ.md

```diff
@@ -21,6 +22,37 @@ io.init_logging(io.LogLevel.Error, 'stderr')
 ```
 You can also enable [CloudWatch logging](https://docs.aws.amazon.com/iot/latest/developerguide/cloud-watch-logs.html) for IoT which will provide you with additional information that is not available on the client side sdk.
 
+### Installation Issues
+
+`awsiotsdk` depends on [awscrt](https://github.com/awslabs/aws-crt-python), which makes use of C extensions. Precompiled wheels are downloaded when installing on major platforms (Mac, Windows, Linux, Raspberry Pi OS). If wheels are unavailable for your platform, your machine must compile some C libraries. For example:
+
+```bash
+# 1. Create a workspace directory to hold all the CRT files
+mkdir crt-workspace
+cd crt-workspace
+
+# 2. Clone the repository, you could select the version you would like to use. You can find the awscrt version used by the current SDK from the file "./aws-iot-device-sdk-python-v2/setup.py". Update the version number in "./aws-iot-device-sdk-python-v2/setup.py" can change the awscrt version you would like to use in awsiotsdk
```

Comment:
Long lines in code snippets are not wrapped, it's somewhat inconvenient to read them.
https://github.com/aws/aws-iot-device-sdk-python-v2/blob/f7638b737f03fbc542018d70557c488777e964e0/documents/FAQ.md#installation-issues

So, I'd split this line and the line below into multiple shorter lines.